### PR TITLE
chore: add Binance cold wallet to known wallets

### PR DESCRIPTION
**Pull request by ItsANameToo**
## Proposed changes

Adds Binance cold wallet to the list of known wallets (it's the wallet they move all their funds to and occasionally put ARK out when they are near 0 in their regular wallet)

## Types of changes

- [x] Other... Please describe: chore

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: resources/views/components/home/tabs.blade.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div x-data="{
     tabsOpen: false,
-    selected: 'transactions',
+    selected: 'blocks',
```
Comment (ItsANameToo) [inline resources/views/components/home/tabs.blade.php]: sus

File: resources/views/components/tables/mobile/blocks.blade.php [hunk 2]
```diff
@@ -7,8 +7,6 @@
 
             <x-tables.rows.mobile.block-forger :model="$block" />
 
-            <x-tables.rows.mobile.block-height :model="$block" />
```
Comment (ItsANameToo) [inline resources/views/components/tables/mobile/blocks.blade.php]: why do we remove data?

File: resources/views/components/tables/rows/mobile/transaction-count.blade.php [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <div>
     @lang('labels.transaction_count')
 
-    {{ $model->transactionCount() }}
+    <div class="inline">
```
Comment (ItsANameToo) [inline resources/views/components/tables/rows/mobile/transaction-count.blade.php]: use a `span` if you want something `inline`

File: resources/views/components/home/tabs.blade.php (hunk 1 above)
Comment (Highjhacker) [reply resources/views/components/home/tabs.blade.php]: Oh yeah forgot that one

File: resources/views/components/tables/mobile/blocks.blade.php (hunk 2 above)
Comment (Highjhacker) [reply resources/views/components/tables/mobile/blocks.blade.php]: Not in the designs

Comment (ItsANameToo) [reply resources/views/components/tables/mobile/blocks.blade.php]: then design is off

Comment (Highjhacker) [reply resources/views/components/tables/mobile/blocks.blade.php]: Ah, was mentioned in the card too by @samharperpittam but will add back then :vinigotit:

**Pull request merged**